src/lambda/config/templates.ts

```diff
@@ -250,19 +291,16 @@ export class TemplatesConfigPopulator {
         }
     }
 
-    private isUnexpectedObjectType(nodeType: jsonParser.NodeType): boolean {
-        return nodeType === 'array'
-            || nodeType === 'property'
-            || nodeType === 'string'
-            || nodeType === 'number'
-            || nodeType === 'boolean'
-    }
-
-    private ensureJsonObjectExists(jsonPath: jsonParser.JSONPath, value: any) {
+    private ensureJsonObjectExists(
```

Comment:
I like the evolution of this method, except now it is no longer tied to `Object` type, consider renaming something like `ensureJsonFieldExists` / `ensureJsonPathExists`?